### ISSUE TITLE
check sy-subrc false positive

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1408]
reproducible in playground

```abap
ASSIGN left_result->data->* TO <left_operand>.
IF <left_operand> IS ASSIGNED.
ENDIF.
```

**Comment by larshp** [comment]
good one 👍
